### PR TITLE
KFSPTS-22312 Add Fields to Tax Output

### PR DESCRIPTION
**Pull request by ajd299**

### CODE REVIEW

**Comment by ajd299** [comment]
Please see my recent commits and let me know if they have addressed all of your feedback

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java [hunk 1]
```diff
@@ -307,6 +309,7 @@ void buildTransactionRows(ResultSet rs, PreparedStatement insertStatement, T sum
             insertStatement.setString(detailRow.chartCode.index - offset, rs.getString(dvRow.chartOfAccountsCode.index));
             insertStatement.setString(detailRow.accountNumber.index - offset, rs.getString(dvRow.accountNumber.index));
             insertStatement.setString(detailRow.paymentReasonCode.index - offset, rs.getString(dvRow.disbVchrPaymentReasonCode.index));
+            insertStatement.setString(detailRow.ledgerDocumentTypeCode.index - offset, getLedgerDocumentTypeCode(rs, dvRow));
             
             insertNullsForTransactionRow(insertStatement, detailRow, offset);
```
[the remark this answers is not in the document]
Comment (ajd299) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java:314]: I have added on to the PRNC builder and created a method in the DV builder.

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java [hunk 2]
```diff
@@ -331,7 +334,38 @@ void buildTransactionRows(ResultSet rs, PreparedStatement insertStatement, T sum
...
+     * </ul>
+     *
+     * @see edu.cornell.kfs.tax.dataaccess.impl.TransactionRowBuilder#insertNullsForTransactionRow(java.sql.PreparedStatement,
+     * edu.cornell.kfs.tax.dataaccess.impl.TaxTableRow.TransactionDetailRow, int)
+     */
+    @Override
+    void insertNullsForTransactionRow(PreparedStatement insertStatement, TransactionDetailRow detailRow, int offset) throws SQLException {
+        super.insertNullsForTransactionRow(insertStatement, detailRow, offset);
+        insertStatement.setString(detailRow.disbursementNbr.index - offset, null);
+        insertStatement.setString(detailRow.paymentStatusCode.index - offset, null);
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java:352]: The DV Builder doesn't appear to explicitly handle the new disbursementTypeCode field. If it's supposed to be null for DVs, then please add it to the insert-nulls method and update the method's Javadocs accordingly.

**Review by cah292 — CHANGES_REQUESTED** [review]
I noticed one more small item that should be addressed, but otherwise the changes look good.

File: src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java (hunk 2 above)
Comment (ajd299) [reply src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java]: Please see my latest commits

**Review by cah292 — APPROVED** [review]
Code changes look good, unit tests pass, my concerns have been addressed, and the related SQL PR is also good to go. Merging.

**Pull request merged**